fiware-region-sanity-tests/docker/README.md

```diff
@@ -0,0 +1,51 @@
+
+# How to use sanity-test with Docker
+
+There are several options to use dockerfiles in this project:
+
+- _"Develop phonehome server"_. See Section **1. Phonehome**.
+- _"Run sanity-tests"_. See Section **2. Sanity-tests**.
+
+
+You do not need to do all of them, just use only the solution for your operations.
+
+You need to have docker in your machine. See the [documentation](https://docs.docker.com/installation/) on how to do this. Additionally, you can use the proper FIWARE Lab docker
+functionality to deploy dockers image there. See the [documentation](https://docs.docker.com/installation/)
+
+
+----
+## 1. Phonehome
+
+Docker allows you to deploy an instance of the phonehome server in order to develop/debug with the service connected to [dbus](https://www.freedesktop.org/wiki/Software/dbus/).
+Follow these steps:
+
+1. Download [fiware-health' source code](https://github.com/telefonicaid/fiware-health) from GitHub (`git clone https://github.com/telefonicaid/fiware-health.git`)
+2. `cd fiware-health/fiware-region-sanity-tests`
+3. Run `docker build -t phonehome .`
+4. Run `docker run -p 8081:8081 -d phonehome`
+
+
```

Comment:
:) True, I know, the question if everyone know it. I want to write a sentence there just saying that to obtain the IP just execute the following command ...